src/planscape/planning/tests.py

```diff
@@ -37,6 +37,34 @@ def setUp(self):
         self.user.save()
         self.geometry = {'features': [
                 {'geometry': {'type': 'Polygon', 'coordinates': [[[1, 2], [2, 3], [3, 4], [1, 2]]]}}]}
+        self.multipolygon_geometry =  {'features': [
+                {'geometry': {'type': 'MultiPolygon', 'coordinates': [[[[1, 2], [2, 3], [3, 4], [1, 2]]]]}}]}
+
+    def test_create_plan(self):
+        self.client.force_login(self.user)
+        response = self.client.post(
+            reverse('planning:create_plan'),
+            {'name': 'test plan', 'region_name': 'Sierra Nevada', 'geometry': self.geometry},
+            content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+        plans = PlanningArea.objects.all()
+        self.assertEqual(plans.count(), 1)
+        plan = plans.first()
+        assert plan is not None
+        self.assertEqual(plan.region_name, 'sierra_cascade_inyo')
+
+    def test_create_plan_multipolygon(self):
+        self.client.force_login(self.user)
+        response = self.client.post(
+            reverse('planning:create_plan'),
+            {'name': 'plan', 'region_name': 'Southern California', 'geometry': self.multipolygon_geometry},
+            content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+        plans = PlanningArea.objects.all()
+        self.assertEqual(plans.count(), 1)
+        plan = plans.first()
+        assert plan is not None
+        self.assertEqual(plan.region_name, 'southern_california')
```

Comment:
Should we also validate that the geometry was saved properly here?

Reply:
yes. added.